community/codelabs/docai-ocr/batch_processing.py

```diff
@@ -1,69 +1,32 @@
 # type: ignore[1]
+# pylint: skip-file
 """
 Makes a Batch Processing Request to Document AI
+Creates request with specific documents
 """
 import re
 from typing import List
 
+from google.api_core.client_options import ClientOptions
 from google.cloud import documentai_v1 as documentai
 from google.cloud import storage
 
 
-def get_documents_from_gcs(
-    gcs_output_uri: str, operation_name: str
-) -> List[documentai.Document]:
-    """
-    Download the document output from GCS.
-    """
-
-    # The GCS API requires the bucket name and URI prefix separately
-    match = re.match(r"gs://([^/]+)/(.+)", gcs_output_uri)
-    output_bucket = match.group(1)
-    prefix = match.group(2)
-
-    # Output files will be in a new subdirectory with Operation ID as the name
-    operation_id = re.search(r"operations\/(\d+)", operation_name, re.IGNORECASE).group(
-        1
-    )
-
-    output_directory = f"{prefix}/{operation_id}"
-
-    storage_client = storage.Client()
-
-    # List of all of the files in the directory
-    # `gs://gcs_output_uri/operation_id`
-    blob_list = list(storage_client.list_blobs(output_bucket, prefix=output_directory))
-
-    output_documents = []
-
-    for blob in blob_list:
-        # Document AI should only output JSON files to GCS
-        if ".json" in blob.name:
-            output_document = documentai.types.Document.from_json(
-                blob.download_as_bytes()
-            )
-            output_documents.append(output_document)
-        else:
-            print(f"Skipping non-supported file type {blob.name}")
-
-    return output_documents
-
-
 PROJECT_ID = "YOUR_PROJECT_ID"
 LOCATION = "YOUR_PROJECT_LOCATION"  # Format is 'us' or 'eu'
 PROCESSOR_ID = "YOUR_PROCESSOR_ID"  # Create processor in Cloud Console
 
 # Format 'gs://input_bucket/directory/file.pdf'
-GCS_INPUT_URI = "INPUT_BUCKET_URI"
+GCS_INPUT_URI = "gs://cloud-samples-data/documentai/codelabs/ocr/Winnie_the_Pooh.pdf"
```

Comment:
will someone be able to access this gcs bucket ?

Reply:
Yes, this is fully publicly accessible (`allUsers` has read permission)